lib/scan_result.dart

```diff
@@ -32,14 +44,31 @@ class ScanResult {
         advertisementData = AdvertisementData._fromJson(json);
 }
 
+/// A container for BLE advertisement data with convenient accessors for
+/// common items.
```

Comment:
?

Reply:
Isn't it true? 😅

Reply:
It is, it just doesn't have much value. 
Would `A container for data with convenient accessors for common items.` be helpful to you if you saw it in a library's documentation?